packages/pyright-internal/src/common/realFileSystem.ts

```diff
@@ -62,7 +66,7 @@ function getArchivePart(path: string) {
 }
 
 function hasZipOrEggExtension(p: string): boolean {

```

Comment:
Thoughts on if/how to change the names of `hasZipOrEggExtension` and `isInZipOrEgg`?

`hasZipOrEggOrJarExtension` seems silly. I'm leaning towards removing the `OrEgg` part. Or just leaving the names as is.

Reply:
My vote is remote the 'orEgg' part.

`hasZiplikeExtension`?